src/__tests__/changemakers.int.test.ts

```diff
@@ -262,29 +272,53 @@ describe('/changemakers', () => {
 		});
 
 		describe('tests that find gold data among proposals and share a common db', () => {
-			it('returns the latest valid value for a base field when auth id is sent', async () => {
-				const systemSource = await loadSystemSource();
-				const systemUser = await loadSystemUser();
+			let systemSource: Source;
+			let systemUser: User;
+			let baseFieldEmail: BaseField;
+			let baseFieldPhone: BaseField;
+			let firstChangemaker: Changemaker;
+			let secondChangemaker: Changemaker;
+			let firstFunder: Funder;
+			let firstFunderOpportunity: Opportunity;
 
-				// Make a base field associated with one opportunity/org, and we'll make three responses.
-				const baseFieldId = (
-					await createBaseField({
-						label: 'Fifty one fifty three',
-						shortCode: 'fifty_one_fifty_three',
-						description: 'Five thousand one hundred fifty three.',
-						dataType: BaseFieldDataType.EMAIL,
-						scope: BaseFieldScope.ORGANIZATION,
-					})
-				).id;
-				// To advance test changemaker IDs beyond 1-2.
-				await insertTestChangemakers();
-				// This should be ID 3, although unfortunately implicitly, and only when tests run in serial.
-				const changemakerId = (
-					await createChangemaker({
-						name: 'Five thousand one hundred forty seven reasons',
-						taxId: '05119',
-					})
-				).id;
+			beforeEach(async () => {
```

Comment:
This is great! I like this approach; good idea, @bickelj.

Reply:
Thanks.